config/projects.yml

```diff
@@ -10,7 +10,7 @@ admin-bundle:
     3.x:
       php: ['5.6', '7.0', '7.1', '7.2']
       versions:
-        symfony: ['2.8', '3.3', '3.4']
+        symfony: ['3.3', '3.4']
```

Comment:
There is no active support for symfony 3.3 anymore: https://symfony.com/roadmap/3.3

Reply:
Lets do it in a following PR, and see if everything will work here 😄

Reply:
already done in this PR 👍